tensorflow/compiler/xla/service/gpu/tests/gpu_kernel_tiling_test.cc

```diff
@@ -815,12 +815,14 @@ TEST_F(GpuKernelTilingTest, RowReductionCorrectShmemUsage) {
   )";
   auto hlo_module = ParseAndReturnVerifiedModule(kHloString).value();
   auto expected_ir = is_built_with_rocm_ ? R"(
-; CHECK: initial_value_addr = internal unnamed_addr addrspace({{[0-9]*}}) global [1024 x float] poison, align 4
+; CHECK: %AMD_FUSION_LDS.t = type { [1 x [1 x [2 x float]]] }
```

Comment:
Can we have platform-independent strings inside the test?

Reply:
> Can we have platform-independent strings inside the test?

Do you mean replace AMD_FUSION_LDS to FUSION_LDS ?

Reply:
Do you mean change AMD_FUSION_LDS to FUSION_LDS ?

Reply:
Hi @cheshire Any update on this PR? Please. Thank you!

Reply:
@i-chaochen Yes.

Reply:
Thanks for the confirm, it's done!